x-pack/platform/plugins/shared/lens/public/lens_attribute_service.ts

```diff
@@ -78,6 +80,29 @@ export function getLensAttributeService(
       managed: boolean;
     }> => {
       const { meta, item } = await savedObjectStore.load(savedObjectId);
+      const configBuilder = new LensConfigBuilder(
```

Comment:
this changes are only to allow me to easy test this with every config i build in lens.